### PR TITLE
ci: add package builder action

### PR DESCRIPTION
**Pull request by PandaGO13**
- Add package builder action
- Use new action on packages publishing workflow

### CODE REVIEW

File: .github/actions/package-builder/action.yml [hunk 1]
```diff
@@ -0,0 +1,48 @@
+name: 'Package Builder'
+description: 'Upload version and build package. Upload build files'
+
+inputs:
+  package:
+    description: 'The package name for build'
+    required: true
+  updateSymbols:
+    description: 'If required update Symbols version'
```
Comment (c1sar) [inline .github/actions/package-builder/action.yml:9]: We can add more detail as: "Updates Symbols dependency in components package.json"

File: .github/actions/package-builder/action.yml [hunk 2]
```diff
@@ -0,0 +1,48 @@
+name: 'Package Builder'
+description: 'Upload version and build package. Upload build files'
```
Comment (c1sar) [inline .github/actions/package-builder/action.yml:2]: We can add more detail as: "Creates a and uploads a build artifact updating the version of the package and its dependencies"

File: .github/workflows/packages-publishing.yml [hunk 3]
```diff
@@ -151,32 +111,11 @@ jobs:
...
-      - name: Upload changes to the repository
-        run: |
-          git config --global user.name "Dynamic bot"
-          git config --global user.email "info@dynamicdevs.io"
-          git push
-        env:
-          github-token: ${{ secrets.GITHUB_TOKEN }}
-      - name: Upload build files
-        uses: actions/upload-artifact@v3
+      - name: Package Builder
```
Comment (c1sar) [inline .github/workflows/packages-publishing.yml:114]: "React package builder"

File: .github/workflows/packages-publishing.yml [hunk 4]
```diff
@@ -92,32 +73,11 @@ jobs:
...
```
Comment (c1sar) [inline .github/workflows/packages-publishing.yml:76]: "Stencil package builder"

File: .github/workflows/packages-publishing.yml [hunk 5]
```diff
@@ -36,29 +36,10 @@ jobs:
...
```
Comment (c1sar) [inline .github/workflows/packages-publishing.yml:39]: "Stencil package builder"

**Review by c1sar — CHANGES_REQUESTED** [review]

File: .github/actions/package-builder/action.yml (hunk 1 above)
Comment (PandaGO13) [reply .github/actions/package-builder/action.yml]: Done

File: .github/workflows/packages-publishing.yml [hunk 6]
```diff
@@ -36,29 +36,10 @@ jobs:
...
+        uses: ./.github/actions/package-builder
         with:
-          name: stencil-component
-          path: dist/libs/stencil-component
+          package: 'stencil-component'
```
[the remark this answers is not in the document]
Comment (PandaGO13) [reply .github/workflows/packages-publishing.yml]: Are not required

File: .github/workflows/packages-publishing.yml (hunk 5 above)
Comment (PandaGO13) [reply .github/workflows/packages-publishing.yml]: Done

**Review by c1sar — APPROVED** [review]

**Pull request merged**